--- ../aquantia-atlantic-kmod.orig/distinfo	2024-07-31 09:30:47.440724000 -0400
+++ distinfo	2024-08-10 15:05:26.925238000 -0400
@@ -1,3 +1,3 @@
-TIMESTAMP = 1683052703
-SHA256 (Aquantia-aqtion-freebsd-0.0.5-c61d27b_GH0.tar.gz) = 586015dad6ee4898bc640016d0c352f2cd3655a91f02597550882ae4c8c73a3b
-SIZE (Aquantia-aqtion-freebsd-0.0.5-c61d27b_GH0.tar.gz) = 61538
+TIMESTAMP = 1723316753
+SHA256 (SheaSmith-aqtion-freebsd-0.0.5-c768889_GH0.tar.gz) = 14711a3d3ee6958043324146a1e545822934ea6e29f4c1153a21022e38b537c1
+SIZE (SheaSmith-aqtion-freebsd-0.0.5-c768889_GH0.tar.gz) = 68478
