--- /dev/null	2022-09-15 11:39:11.807833000 +0000
+++ files/patch-sdl	2022-09-15 11:38:09.568945000 +0000
@@ -0,0 +1,11 @@
+--- configure.orig	2022-09-08 10:22:56 UTC
++++ configure
+@@ -6549,7 +6549,7 @@ fi
+ 
+ if enabled sdl2; then
+     SDL2_CONFIG="${cross_prefix}sdl2-config"
+-    test_pkg_config sdl2 "sdl2 >= 2.0.1 sdl2 < 2.1.0" SDL_events.h SDL_PollEvent
++    test_pkg_config sdl2 "sdl2 >= 2.0.1 sdl2 < 3.0.0" SDL_events.h SDL_PollEvent
+     if disabled sdl2 && "${SDL2_CONFIG}" --version > /dev/null 2>&1; then
+         sdl2_cflags=$("${SDL2_CONFIG}" --cflags)
+         sdl2_extralibs=$("${SDL2_CONFIG}" --libs)
