--- ../mixxx.orig/pkg-plist	2023-12-07 16:00:33.463836000 +0000
+++ pkg-plist	2023-12-31 03:39:47.614109000 +0000
@@ -1,5 +1,9 @@
 bin/mixxx
 share/applications/org.mixxx.Mixxx.desktop
+%%PORTDOCS%%%%DOCSDIR%%/COPYING
+%%PORTDOCS%%%%DOCSDIR%%/LICENSE
+%%PORTDOCS%%%%DOCSDIR%%/Mixxx-Keyboard-Shortcuts.pdf
+%%PORTDOCS%%%%DOCSDIR%%/README.md
 share/icons/hicolor/128x128/apps/mixxx.png
 share/icons/hicolor/256x256/apps/mixxx.png
 share/icons/hicolor/32x32/apps/mixxx.png
@@ -186,6 +190,8 @@
 %%DATADIR%%/controllers/Numark-N4-scripts.js
 %%DATADIR%%/controllers/Numark-NS7-scripts.js
 %%DATADIR%%/controllers/Numark-Omni-Control-scripts.js
+%%DATADIR%%/controllers/Numark-Party-Mix.midi.xml
+%%DATADIR%%/controllers/Numark-Party-Mix.scripts.js
 %%DATADIR%%/controllers/Numark-Total-Control-scripts.js
 %%DATADIR%%/controllers/Numark-V7-scripts.js
 %%DATADIR%%/controllers/Numark-iDJ-Live-II-scripts.js
@@ -245,11 +251,13 @@
 %%DATADIR%%/controllers/TrakProDJ-iPad-scripts.js
 %%DATADIR%%/controllers/Traktor Kontrol F1.hid.xml
 %%DATADIR%%/controllers/Traktor Kontrol S2 MK3.hid.xml
+%%DATADIR%%/controllers/Traktor Kontrol S2 Mk1.hid.xml
 %%DATADIR%%/controllers/Traktor Kontrol S2 Mk2.hid.xml
 %%DATADIR%%/controllers/Traktor Kontrol S3.hid.xml
 %%DATADIR%%/controllers/Traktor Kontrol S4 MK2.hid.xml
 %%DATADIR%%/controllers/Traktor Kontrol X1.midi.xml
 %%DATADIR%%/controllers/Traktor-Kontrol-F1-scripts.js
+%%DATADIR%%/controllers/Traktor-Kontrol-S2-MK1-hid-scripts.js
 %%DATADIR%%/controllers/Traktor-Kontrol-S2-MK2-hid-scripts.js
 %%DATADIR%%/controllers/Traktor-Kontrol-S2-MK3-hid-scripts.js
 %%DATADIR%%/controllers/Traktor-Kontrol-S3-hid-scripts.js
@@ -1421,9 +1429,8 @@
 %%DATADIR%%/skins/LateNight/palemoon/style/spinny_mask_34.svg
 %%DATADIR%%/skins/LateNight/palemoon/style/splitter_handle_horizontal.svg
 %%DATADIR%%/skins/LateNight/palemoon/style/splitter_handle_horizontal_pressed.svg
-%%DATADIR%%/skins/LateNight/palemoon/style/splitter_handle_vertical.png
 %%DATADIR%%/skins/LateNight/palemoon/style/splitter_handle_vertical.svg
-%%DATADIR%%/skins/LateNight/palemoon/style/splitter_handle_vertical_pressed.png
+%%DATADIR%%/skins/LateNight/palemoon/style/splitter_handle_vertical_pressed.svg
 %%DATADIR%%/skins/LateNight/palemoon/style/vinyl_control_0.svg
 %%DATADIR%%/skins/LateNight/palemoon/style/vinyl_control_1.svg
 %%DATADIR%%/skins/LateNight/palemoon/style/vinyl_control_2.svg
@@ -2509,20 +2516,12 @@
 %%DATADIR%%/skins/Tango/graphics/splitterHori_handle_pressed.svg
 %%DATADIR%%/skins/Tango/graphics/splitterVert_handle.svg
 %%DATADIR%%/skins/Tango/graphics/splitterVert_handle_pressed.svg
-%%DATADIR%%/skins/Tango/graphics/vumeter_clipping.png
-%%DATADIR%%/skins/Tango/graphics/vumeter_clipping.svg
-%%DATADIR%%/skins/Tango/graphics/vumeter_clipping_over.png
-%%DATADIR%%/skins/Tango/graphics/vumeter_clipping_over.svg
 %%DATADIR%%/skins/Tango/graphics/vumeter_level.png
 %%DATADIR%%/skins/Tango/graphics/vumeter_level.svg
 %%DATADIR%%/skins/Tango/graphics/vumeter_level_over.png
 %%DATADIR%%/skins/Tango/graphics/vumeter_level_over.svg
-%%DATADIR%%/skins/Tango/graphics/vumeter_mini_clipping.png
-%%DATADIR%%/skins/Tango/graphics/vumeter_mini_clipping.svg
 %%DATADIR%%/skins/Tango/graphics/vumeter_mini_clipping_over.png
-%%DATADIR%%/skins/Tango/graphics/vumeter_mini_clipping_over.svg
 %%DATADIR%%/skins/Tango/graphics/vumeter_mini_level.png
-%%DATADIR%%/skins/Tango/graphics/vumeter_mini_level.svg
 %%DATADIR%%/skins/Tango/graphics/vumeter_mini_level_over.png
 %%DATADIR%%/skins/Tango/graphics/vumeter_mini_level_over.svg
 %%DATADIR%%/skins/Tango/helpers/pfl_cover.xml
