--- /dev/null	2023-03-26 16:37:21.271706000 +0000
+++ files/extra-patch-waflib_ConfigSet.py	2023-03-26 16:36:42.625980000 +0000
@@ -0,0 +1,11 @@
+--- waflib/ConfigSet.py.orig	2023-03-26 16:35:12.155685000 +0000
++++ waflib/ConfigSet.py	2023-03-26 16:36:05.363339000 +0000
+@@ -146,7 +146,7 @@
+ 		Utils.writef(filename,''.join(buf))
+ 	def load(self,filename):
+ 		tbl=self.table
+-		code=Utils.readf(filename,m='rU')
++		code=Utils.readf(filename,m='r')
+ 		for m in re_imp.finditer(code):
+ 			g=m.group
+ 			tbl[g(2)]=eval(g(3))
