--- /dev/null	2023-03-26 16:24:49.893920000 +0000
+++ files/extra-patch-waflib_Context.py	2023-03-26 16:09:41.593012000 +0000
@@ -0,0 +1,20 @@
+--- waflib/Context.py.orig	2023-03-26 16:02:05.671555000 +0000
++++ waflib/Context.py	2023-03-26 16:06:21.635021000 +0000
+@@ -106,7 +106,7 @@
+ 				cache[node]=True
+ 				self.pre_recurse(node)
+ 				try:
+-					function_code=node.read('rU',encoding)
++					function_code=node.read('r',encoding)
+ 					exec(compile(function_code,node.abspath(),'exec'),self.exec_dict)
+ 				finally:
+ 					self.post_recurse(node)
+@@ -346,7 +346,7 @@
+ 		pass
+ 	module=imp.new_module(WSCRIPT_FILE)
+ 	try:
+-		code=Utils.readf(path,m='rU',encoding=encoding)
++		code=Utils.readf(path,m='r',encoding=encoding)
+ 	except EnvironmentError:
+ 		raise Errors.WafError('Could not read the file %r'%path)
+ 	module_dir=os.path.dirname(path)
