--- ../mixxx.orig/distinfo	2023-02-26 02:40:38.031614000 +0000
+++ ./distinfo	2023-03-27 15:48:38.396917000 +0000
@@ -1,3 +1,3 @@
-TIMESTAMP = 1646682590
-SHA256 (mixxxdj-mixxx-2.3.2_GH0.tar.gz) = b6a0ed80bfbda5b65014e75ffd132f8a7474b260ef18d1e4e2c203e21bb28695
-SIZE (mixxxdj-mixxx-2.3.2_GH0.tar.gz) = 40138947
+TIMESTAMP = 1697509857
+SHA256 (mixxxdj-mixxx-2.3.6_GH0.tar.gz) = 0030d07c1506ccc13daa63d851921381b5bf838e9407cd666557d951ac093c52
+SIZE (mixxxdj-mixxx-2.3.6_GH0.tar.gz) = 41285084
