--- ../cassandra4.orig/files/patch-src_java_org_apache_cassandra_config_Config.java	2023-03-26 22:51:26.979412000 +0000
+++ ./files/patch-src_java_org_apache_cassandra_config_Config.java	2023-03-26 23:38:03.320947000 +0000
@@ -1,6 +1,6 @@
---- src/java/org/apache/cassandra/config/Config.java.orig	2020-08-28 13:55:55 UTC
+--- src/java/org/apache/cassandra/config/Config.java.orig	2022-05-06 16:40:06 UTC
 +++ src/java/org/apache/cassandra/config/Config.java
-@@ -290,7 +290,7 @@ public class Config
+@@ -304,7 +304,7 @@ public class Config
      public volatile int key_cache_save_period = 14400;
      public volatile int key_cache_keys_to_save = Integer.MAX_VALUE;
  
