--- ../mongodb70.orig/files/extrapatch-.._spidermonkey-82aac6af18abcd5bf188afbc821779ccb0ca0902_python_mozbuild_mozbuild_action_process_define_files.py	1969-12-31 19:00:00.000000000 -0500
+++ ./files/extrapatch-.._spidermonkey-82aac6af18abcd5bf188afbc821779ccb0ca0902_python_mozbuild_mozbuild_action_process_define_files.py	2023-08-31 13:23:00.210545000 -0400
@@ -0,0 +1,11 @@
+--- ../spidermonkey-82aac6af18abcd5bf188afbc821779ccb0ca0902/python/mozbuild/mozbuild/action/process_define_files.py.orig	2023-08-31 13:21:36.435605000 -0400
++++ ../spidermonkey-82aac6af18abcd5bf188afbc821779ccb0ca0902/python/mozbuild/mozbuild/action/process_define_files.py	2023-08-31 13:21:52.046313000 -0400
+@@ -36,7 +36,7 @@
+     ) and not config.substs.get("JS_STANDALONE"):
+         config = PartialConfigEnvironment(mozpath.join(topobjdir, "js", "src"))
+ 
+-    with open(path, "rU") as input:
++    with open(path, "r") as input:
+         r = re.compile(
+             "^\s*#\s*(?P<cmd>[a-z]+)(?:\s+(?P<name>\S+)(?:\s+(?P<value>\S+))?)?", re.U
+         )
