--- ../cassandra4.orig/files/patch-src_java_org_apache_cassandra_service_StartupChecks.java	2023-03-26 22:51:26.979613000 +0000
+++ ./files/patch-src_java_org_apache_cassandra_service_StartupChecks.java	2023-03-26 23:38:03.321339000 +0000
@@ -1,4 +1,4 @@
---- src/java/org/apache/cassandra/service/StartupChecks.java.orig	2020-08-28 13:55:56 UTC
+--- src/java/org/apache/cassandra/service/StartupChecks.java.orig	2021-03-26 19:57:28 UTC
 +++ src/java/org/apache/cassandra/service/StartupChecks.java
 @@ -281,7 +281,14 @@ public class StartupChecks
      {
