--- ../cassandra4.orig/files/cassandra.in	2023-03-26 22:51:26.978792000 +0000
+++ ./files/cassandra.in	2023-03-26 23:38:03.319409000 +0000
@@ -23,12 +23,11 @@
 : ${cassandra_enable:=NO}
 : ${cassandra_user:=cassandra}
 
-command=%%PREFIX%%/bin/cassandra
-command_args="-p /var/run/cassandra/cassandra.pid >/dev/null 2>/dev/null"
-start_precmd="${name}_prestart"
-
-procname=%%JAVA_HOME%%/bin/java
 pidfile=/var/run/cassandra/cassandra.pid
+
+command=/usr/sbin/daemon
+command_args="-frP ${pidfile} %%PREFIX%%/bin/cassandra -f"
+start_precmd="${name}_prestart"
 
 cassandra_prestart()
 {
