--- ../wasmer.orig/files/patch-lib_compiler-llvm_src_translator_intrinsics.rs	1970-01-01 00:00:00.000000000 +0000
+++ ./files/patch-lib_compiler-llvm_src_translator_intrinsics.rs	2024-06-19 10:27:40.154110000 +0000
@@ -0,0 +1,11 @@
+--- lib/compiler-llvm/src/translator/intrinsics.rs.orig	2024-06-19 10:26:02.313989000 +0000
++++ lib/compiler-llvm/src/translator/intrinsics.rs	2024-06-19 10:26:26.636672000 +0000
+@@ -4,6 +4,8 @@
+ //!
+ //! [llvm-intrinsics]: https://llvm.org/docs/LangRef.html#intrinsic-functions
+ 
++#![allow(dead_code)]
++
+ use crate::abi::Abi;
+ use inkwell::values::BasicMetadataValueEnum;
+ use inkwell::{
