--- ../gnucash.orig/Makefile	2023-03-28 16:02:37.380173000 +0000
+++ ./Makefile	2023-03-28 16:01:28.966498000 +0000
@@ -86,8 +86,15 @@
 OFX_LIB_DEPENDS=	libofx.so:finance/libofx
 OFX_VARS=		GLIB_SCHEMAS+=org.gnucash.GnuCash.dialogs.import.ofx.gschema.xml
 PYTHON_CMAKE_BOOL=	WITH_PYTHON
-PYTHON_USES=		python:3.6+
+PYTHON_USES=		python:3.10
 
+.include <bsd.port.pre.mk>
+
+.if ${PORT_OPTIONS:MPYTHON}
+CMAKE_ARGS+=		-DPYTHON_EXECUTABLE:STRING=${PYTHON_CMD} \
+			-DPYTHON_INCLUDEDIR:STRING=${PYTHON_INCLUDEDIR}
+.endif
+
 post-install-PYTHON-on:
 	@(cd ${STAGEDIR}${PREFIX} \
 		&& ${PYTHON_CMD} ${PYTHON_LIBDIR}/compileall.py \
@@ -97,4 +104,4 @@
 		-d ${PYTHONPREFIX_SITELIBDIR} -f ${PYTHONPREFIX_SITELIBDIR:S;${PREFIX}/;;}/${PORTNAME})
 	@${STRIP_CMD} ${STAGEDIR}${PYTHON_SITELIBDIR}/gnucash/_gnucash_core_c.so
 
-.include <bsd.port.mk>
+.include <bsd.port.post.mk>
