--- ../etlegacy.orig/Makefile	2024-07-31 09:30:43.462567000 -0400
+++ Makefile	2024-08-13 18:49:27.545587000 -0400
@@ -3,8 +3,12 @@
 PORTREVISION=	2
 CATEGORIES=	games
 MASTER_SITES=	https://mirror.etlegacy.com/geoip/:geoip \
+		https://mirror.etlegacy.com/etmain/:etmain \
 		https://mirror.etlegacy.com/wolfadmin/:wolfadmin
 DISTFILES=	wolfadmin.tar.gz:wolfadmin \
+		pak0.pk3:etmain \
+		pak1.pk3:etmain \
+		pak2.pk3:etmain \
 		GeoIP.dat.tar.gz:geoip
 DIST_SUBDIR=	${PORTNAME}-${DISTVERSION}
 EXTRACT_ONLY=	${DISTNAME}${EXTRACT_SUFX}
@@ -53,8 +57,6 @@
 			libcjson.so:devel/libcjson
 DEDICATED_USES=		minizip
 
-SUB_FILES=	pkg-message
-
 .include <bsd.port.pre.mk>
 
 .if ${ARCH} == "i386"
@@ -123,6 +125,9 @@
 	@${MKDIR} ${WRKDIR}/.build/legacy
 	@cd ${WRKDIR}/.build/legacy && ${EXTRACT_CMD} ${EXTRACT_BEFORE_ARGS} ${_DISTDIR}/GeoIP.dat.tar.gz ${EXTRACT_AFTER_ARGS}
 	@cd ${WRKDIR}/.build/legacy && ${EXTRACT_CMD} ${EXTRACT_BEFORE_ARGS} ${_DISTDIR}/wolfadmin.tar.gz ${EXTRACT_AFTER_ARGS}
+
+post-install:
+	${CP} -v ${DISTDIR}/${DIST_SUBDIR}/*.pk3 ${STAGEDIR}${PREFIX}/lib/etlegacy/etmain/
 
 post-patch:
 	${REINPLACE_CMD} -e 's|AMD64|amd64|g' ${WRKSRC}/cmake/ETLPlatform.cmake
