--- ../inkscape.orig/Makefile	2023-03-15 13:39:39.730607000 +0000
+++ Makefile	2023-03-15 13:39:52.103597000 +0000
@@ -43,7 +43,7 @@
 		${PYTHON_PKGNAMEPREFIX}CacheControl>0:www/py-cachecontrol@${PY_FLAVOR} \
 		${PYTHON_PKGNAMEPREFIX}cssselect>0:www/py-cssselect@${PY_FLAVOR} \
 		${PYTHON_PKGNAMEPREFIX}requests>0:www/py-requests@${PY_FLAVOR} \
-		ps2pdf:print/ghostscript9-agpl-base \
+		ps2pdf:print/ghostscript9-base \
 		fig2dev:print/fig2dev
 
 USES=		compiler:c++17-lang cmake cpe desktop-file-utils gnome \
