--- ../opencv.orig/distinfo	
+++ distinfo	
@@ -15,3 +15,5 @@ SHA256 (opencv-opencv_3rdparty-8afa57abc8229d611c4937165d20e2a2d9fc5a12_GH0.tar.
 SIZE (opencv-opencv_3rdparty-8afa57abc8229d611c4937165d20e2a2d9fc5a12_GH0.tar.gz) = 63301261
 SHA256 (opencv-opencv_3rdparty-a56b6ac6f030c312b2dce17430eef13aed9af274_GH0.tar.gz) = 6d3552455d927dd28cdd6a78470398208ff2167b19b0dd51ab30a241d900cb4c
 SIZE (opencv-opencv_3rdparty-a56b6ac6f030c312b2dce17430eef13aed9af274_GH0.tar.gz) = 135706483
+SHA256 (496eed950f6d0e7fd92619d47e3cec8f06e96ace.patch) = 28edbd94c443b6aaa3054b4f9b35a4ad9a89dc45f40d408bcf3036950a072a32
+SIZE (496eed950f6d0e7fd92619d47e3cec8f06e96ace.patch) = 671
