--- ../spidermonkey102.orig/Makefile	2023-02-28 14:27:13.825484000 +0000
+++ Makefile	2023-02-28 14:28:01.988779000 +0000
@@ -13,7 +13,7 @@
 LICENSE=	MPL20
 LICENSE_FILE=	${WRKSRC}/LICENSE
 
-BUILD_DEPENDS=	${LOCALBASE}/bin/python${PYTHON3_DEFAULT}:lang/python${PYTHON3_DEFAULT:S/.//g} \
+BUILD_DEPENDS=	${LOCALBASE}/bin/python3.9:lang/python39 \
 		${RUST_DEFAULT}>=1.35:lang/${RUST_DEFAULT} \
 		autoconf2.13:devel/autoconf2.13 \
 		rust-cbindgen>=0.8.7:devel/rust-cbindgen
@@ -22,7 +22,7 @@
 		libnspr4.so:devel/nspr
 
 USES=		compiler:c++17-lang gmake localbase pathfix pkgconfig \
-		python:build readline tar:xz
+		python:3.9,build readline tar:xz
 USE_LDCONFIG=	yes
 SP_VER=		102
 HAS_CONFIGURE=	yes
