--- ../sbcl.orig/Makefile	2024-07-02 15:44:13.667855000 -0400
+++ Makefile	2024-07-02 15:47:58.880029000 -0400
@@ -147,13 +147,13 @@
 		${WRKSRC}/doc/manual/start-stop.texinfo
 
 do-build:
-	(cd ${WRKSRC} && ${SH} make.sh ${MAKE_SH_ARGS})
+	(cd ${WRKSRC} && ${SETENVI} ${WRK_ENV} ${MAKE_ENV} ${SH} make.sh ${MAKE_SH_ARGS})
 
 post-build-DOCS-on:
 	${DO_MAKE_BUILD} -C ${WRKSRC}/doc/manual info html
 
 do-install:
-	(cd ${WRKSRC} && ${SETENV} \
+	(cd ${WRKSRC} && ${SETENVI} ${WRK_ENV} ${MAKE_ENV} \
 	INSTALL_ROOT="${STAGEDIR}${PREFIX}" \
 	MAN_DIR="${STAGEDIR}${PREFIX}/share/man" \
 	INFO_DIR="${STAGEDIR}${PREFIX}/${INFO_PATH}" \
@@ -181,6 +181,6 @@
 	${RM} -r ${STAGEDIR}${DOCSDIR}/html # empty directory created by install.sh
 
 check regression-test test: build
-	(cd ${WRKSRC}/tests && ${SH} run-tests.sh)
+	(cd ${WRKSRC}/tests && ${SETENVI} ${WRK_ENV} ${MAKE_ENV} ${SH} run-tests.sh)
 
 .include <bsd.port.mk>
