--- ../py-gmpy.orig/Makefile	2023-03-27 10:36:51.106415000 +0000
+++ ./Makefile	2023-03-27 10:32:31.111109000 +0000
@@ -14,8 +14,15 @@
 USES=		python:3.6+ zip
 USE_PYTHON=	distutils autoplist
 
+.include <bsd.port.pre.mk>
+
 post-extract:
 	${REINPLACE_CMD} -e 's#/usr/local#${LOCALBASE}#g' \
 		${WRKSRC}/setup.py
 
-.include <bsd.port.mk>
+.if ${PYTHON_REL} >= 31100
+EXTRA_PATCHES+=	${PATCHDIR}/extra-patch-src_gmpy.c \
+		${PATCHDIR}/extra-patch-src_mpz__pylong.c
+.endif
+
+.include <bsd.port.post.mk>
