--- ../e2fsprogs-libuuid.orig/Makefile	2024-01-24 02:29:23.838317000 +0000
+++ Makefile	2024-06-18 09:55:08.541517000 +0000
@@ -28,14 +28,14 @@
 
 pre-build:
 	${MKDIR} ${WRKSRC}/lib/uuid/elfshared
-	cd ${WRKSRC}/util && ${MAKE_CMD} subst
+	cd ${WRKSRC}/util && ${SETENVI} ${WRK_ENV} ${MAKE_ENV} ${MAKE_CMD} subst
 
 # ulimit guards against runaway tests
 # failure to launch uuidd is fine (one might be running, or we may lack
 # privileges); if it works, it'll quit after 50 seconds
 post-build:
-	cd ${WRKSRC}/misc && ${MAKE_CMD} uuidgen uuidgen.1 uuidd uuidd.8
-	cd ${INSTALL_WRKSRC} && ( ulimit -t 5 && ${MAKE_CMD} check )
+	cd ${WRKSRC}/misc && ${SETENVI} ${WRK_ENV} ${MAKE_ENV} ${MAKE_CMD} uuidgen uuidgen.1 uuidd uuidd.8
+	cd ${INSTALL_WRKSRC} && ( ulimit -t 5 && ${SETENVI} ${WRK_ENV} ${TEST_ENV} ${MAKE_CMD} check )
 
 post-install:
 	${INSTALL_PROGRAM} ${WRKSRC}/misc/uuidgen ${STAGEDIR}${PREFIX}/bin/
