--- ../x265.orig/distinfo
+++ distinfo
@@ -1,3 +1,5 @@
-TIMESTAMP = 1600662771
-SHA256 (x265-3.4.tar.gz) = 01f17d94689ef79807fa2b4bd7f5f58bb18416e72a135c7f56c30162a980cfb8
-SIZE (x265-3.4.tar.gz) = 1532396
+TIMESTAMP = 1698027802
+SHA256 (x265-3.5.tar.gz) = 8a5fcfddb3e2db520b637e475a507fb00117d554bdb3204c5828626d888c306a
+SIZE (x265-3.5.tar.gz) = 1538610
+SHA256 (x265-asm-warnings.patch) = 0faa3278887a47ed8c968c994e4cc71a2921aac38f5d560dae898124f806871d
+SIZE (x265-asm-warnings.patch) = 4526
