--- ../mlt7.orig/Makefile
+++ Makefile
@@ -1,6 +1,5 @@
 PORTNAME=		${_MLT_PORTNAME}
 DISTVERSION=		${_MLT_VERSION}
-PORTREVISION=	3
 CATEGORIES?=		multimedia
 # Don't use GitHub "fake" downloads that are auto-generated from a
 # tag; the project uploads tarballs for each release.
@@ -9,10 +8,6 @@ MASTER_SITES=		https://github.com/mltframework/mlt/releases/download/v${DISTVERS
 DISTFILES=		mlt-${DISTVERSION}${EXTRACT_SUFX}
 WRKSRC=			${WRKDIR}/mlt-${DISTVERSION}
 
-PATCH_SITES=		https://github.com/mltframework/mlt/commit/
-PATCHFILES=		a3d49ccb92e2544bcc1957ce046bf63f71b468a6.patch:-p1 \
-			1eee2187410ae16f520409dcdcf6de789d8bafd9.patch:-p1
-
 MAINTAINER=		kde@FreeBSD.org
 COMMENT?=		Multimedia framework for TV broadcasting
 WWW=		https://www.mltframework.org/
@@ -109,7 +104,8 @@ PLUSGPL_DESC=		Enable plus GPL module
 PLUSGPL_CMAKE_BOOL=	MOD_PLUSGPL
 
 PLUS_DESC=		Enable plus module
-PLUS_LIB_DEPENDS=	libfftw3.so:math/fftw3
+PLUS_LIB_DEPENDS=	libfftw3.so:math/fftw3 \
+			libebur128.so:audio/ebur128
 PLUS_CMAKE_BOOL=	MOD_PLUS
 
 RESAMPLE_DESC=		Enable resample module
