--- ../../UIDs.orig	2023-04-08 04:09:51.877777000 +0000
+++ ../../UIDs	2023-04-08 04:13:20.885716000 +0000
@@ -722,7 +722,7 @@
 # free: 773
 # free: 774
 # free: 775
-# free: 776
+namecoin:*:776:776::0:0:namecoin Daemon:/var/db/namecoin:/usr/sbin/nologin
 openxpki:*:777:777::0:0:OpenXPKI Owner:/nonexistent:/usr/sbin/nologin
 litecoin:*:778:778::0:0:litecoin Daemon:/var/db/litecoin:/usr/sbin/nologin
 bitcoin:*:779:779::0:0:bitcoin Daemon:/var/db/bitcoin:/usr/sbin/nologin
