--- ../../UIDs.orig
+++ ../../UIDs
@@ -730,7 +730,7 @@ zetacoin:*:780:780::0:0:ZetaCoin Daemon:/nonexistent:/usr/sbin/nologin
 monero:*:781:781::0:0:Monero Daemon:/var/db/monero:/usr/sbin/nologin
 zeek:*:782:782::0:0:Zeek Network Security Monitor:/nonexistent:/usr/sbin/nologin
 bitcoincash:*:783:783::0:0:Bitcoin Cash Daemon:/var/db/bitcoincash:/usr/sbin/nologin
-# free: 784
+bitcoinsv:*:784:784::0:0:Bitcoin SV Daemon:/var/db/bitcoinsv:/usr/sbin/nologin
 bitcoingold:*:785:785::0:0:Bitcoin Gold Daemon:/var/db/bitcoingold:/usr/sbin/nologin
 # free: 786
 # free: 787
