--- /dev/null	2023-02-23 17:04:28.904112000 +0000
+++ files/patch-src_rpc_names.cpp	2023-02-23 16:55:09.979422000 +0000
@@ -0,0 +1,10 @@
+--- src/rpc/names.cpp.orig	2023-02-23 16:44:43.398660000 +0000
++++ src/rpc/names.cpp	2023-02-23 16:44:58.322833000 +0000
+@@ -20,6 +20,7 @@
+ #include <rpc/server_util.h>
+ #include <script/names.h>
+ #include <txmempool.h>
++#include <util/system.h>
+ #include <util/strencodings.h>
+ #include <validation.h>
+ #ifdef ENABLE_WALLET
