--- ../namecoin.orig/Makefile
+++ Makefile
@@ -1,5 +1,5 @@
 PORTNAME=	namecoin
 DISTVERSIONPREFIX=	nc
-DISTVERSION=	24.0
+DISTVERSION=	25.1
 PORTREVISION=	4
 PORTEPOCH=	1
@@ -75,12 +75,12 @@ QRCODES_LIB_DEPENDS=	libqrencode.so:graphics/libqrencode
 
 WALLET_CONFIGURE_OFF=	--disable-wallet
 WALLET_CONFIGURE_ON=	--enable-wallet \
-			--with-sqlite=no \
+			--with-sqlite=yes \
 			--with-incompatible-bdb \
 			BDB_CFLAGS=-I${BDB_INCLUDE_DIR} \
 			BDB_LIBS="-L${BDB_LIB_DIR} \
 			-Wl,-rpath=${BDB_LIB_DIR} -l${BDB_LIB_CXX_NAME}"
-WALLET_USES=		bdb:5+
+WALLET_USES=		bdb:5+ sqlite:3
 
 ZMQ_CONFIGURE_ENABLE=	zmq
 ZMQ_BUILD_DEPENDS=	libzmq4>0:net/libzmq4
