--- ../openldap26-server.orig/Makefile	2024-06-17 10:45:29.264093000 +0000
+++ Makefile	2024-06-18 11:23:38.814081000 +0000
@@ -404,13 +404,13 @@
 
 .if !defined(CLIENT_ONLY)
 test: build
-	@(cd ${BUILD_WRKSRC} && ${SETENV} ${MAKE_ENV} \
+	@(cd ${BUILD_WRKSRC} && ${SETENVI} ${WRK_ENV} ${TEST_ENV} \
 		${MAKE_CMD} ${MAKE_FLAGS} ${MAKEFILE} ${MAKE_ARGS} test)
 
 post-build:
 .for module in ${CONTRIB_MODULES}
 	@(cd ${BUILD_WRKSRC}/contrib/${module} && \
-		${SETENV} ${MAKE_ENV} ${MAKE_CMD} ${MAKE_FLAGS} ${MAKEFILE} \
+		${SETENVI} ${WRK_ENV} ${MAKE_ENV} ${MAKE_CMD} ${MAKE_FLAGS} ${MAKEFILE} \
 		${MAKE_ARGS} prefix="${PREFIX}" all)
 .endfor
 .endif
@@ -438,7 +438,7 @@
 	${MKDIR} ${STAGEDIR}${LDAP_RUN_DIR}
 .for module in ${CONTRIB_MODULES}
 	(cd ${WRKSRC}/contrib/${module} && \
-		${SETENV} ${MAKE_ENV} ${MAKE_CMD} ${MAKE_FLAGS} ${MAKEFILE} \
+		${SETENVI} ${WRK_ENV} ${MAKE_ENV} ${MAKE_CMD} ${MAKE_FLAGS} ${MAKEFILE} \
 		${MAKE_ARGS} prefix="${PREFIX}" install)
 .endfor
 .for cleanfile in ${CONTRIB_CLEANFILES}
