--- ../aquantia-atlantic-kmod.orig/files/patch-aq__main.c	2024-07-31 09:30:47.441025000 -0400
+++ files/patch-aq__main.c	2024-08-10 15:25:26.842398000 -0400
@@ -1,15 +1,26 @@
---- aq_main.c.orig	2019-09-24 14:45:34 UTC
-+++ aq_main.c
-@@ -193,8 +193,12 @@ static driver_t aq_driver = {
- 	"aq", aq_methods, sizeof(struct aq_dev),
- };
+--- aq_main.c.bak	2024-08-10 15:19:35.828433000 -0400
++++ aq_main.c	2024-08-10 15:24:38.020480000 -0400
+@@ -110,6 +110,7 @@
+ #define AQ_DEVICE_ID_AQC112S	0x92B1
  
-+#if __FreeBSD_version >= 1400058
-+DRIVER_MODULE(atlantic, pci, aq_driver, 0, 0);
-+#else
- static devclass_t aq_devclass;
- DRIVER_MODULE(atlantic, pci, aq_driver, aq_devclass, 0, 0);
-+#endif
+ #define AQ_DEVICE_ID_AQC113	0x14C0
++#define AQ_DEVICE_ID_AQC113CS	0x94C0
  
- MODULE_DEPEND(atlantic, pci, 1, 1, 1);
- MODULE_DEPEND(atlantic, ether, 1, 1, 1);
+ static pci_vendor_info_t aq_vendor_info_array[] = {
+ 	PVID(AQUANTIA_VENDOR_ID, AQ_DEVICE_ID_0001, "Aquantia AQtion 10Gbit Network Adapter"),
+@@ -133,6 +134,7 @@
+ 	PVID(AQUANTIA_VENDOR_ID, AQ_DEVICE_ID_AQC112S, "Aquantia AQtion 2.5Gbit Network Adapter"),
+ 
+ 	PVID(AQUANTIA_VENDOR_ID, AQ_DEVICE_ID_AQC113, "Aquantia AQtion 10Gbit Network Adapter"),
++	PVID(AQUANTIA_VENDOR_ID, AQ_DEVICE_ID_AQC113CS, "Aquantia AQtion 10Gbit Network Adapter"),
+ 
+ 	PVID_END
+ };
+@@ -1165,6 +1167,7 @@
+ 	case AQ_DEVICE_ID_AQC107:
+ 	case AQ_DEVICE_ID_AQC107S:
+ 	case AQ_DEVICE_ID_AQC113:
++	case AQ_DEVICE_ID_AQC113CS:
+ 		softc->media_type = AQ_MEDIA_TYPE_TP;
+ 		softc->link_speeds = AQ_LINK_ALL;
+ 		break;
