--- ../libpwquality.orig/pkg-plist	2022-09-15 21:24:30.445789000 +0000
+++ pkg-plist	2022-09-15 21:23:32.360160000 +0000
@@ -6,7 +6,12 @@
 lib/libpwquality.so
 lib/libpwquality.so.1
 lib/libpwquality.so.1.0.2
-%%PYTHON%%%%PYTHON_SITELIBDIR%%/pwquality-1.4.4-py%%PYTHON_VER%%.egg-info
+%%PYTHON%%%%PYTHON_SITELIBDIR%%/pwquality-1.4.4-py%%PYTHON_VER%%.egg-info/PKG-INFO
+%%PYTHON%%%%PYTHON_SITELIBDIR%%/pwquality-1.4.4-py%%PYTHON_VER%%.egg-info/SOURCES.txt
+%%PYTHON%%%%PYTHON_SITELIBDIR%%/pwquality-1.4.4-py%%PYTHON_VER%%.egg-info/dependency_links.txt
+%%PYTHON%%%%PYTHON_SITELIBDIR%%/pwquality-1.4.4-py%%PYTHON_VER%%.egg-info/native_libs.txt
+%%PYTHON%%%%PYTHON_SITELIBDIR%%/pwquality-1.4.4-py%%PYTHON_VER%%.egg-info/not-zip-safe
+%%PYTHON%%%%PYTHON_SITELIBDIR%%/pwquality-1.4.4-py%%PYTHON_VER%%.egg-info/top_level.txt
 %%PYTHON%%%%PYTHON_SITELIBDIR%%/pwquality%%PYTHON_EXT_SUFFIX%%.so
 libdata/pkgconfig/pwquality.pc
 man/man1/pwmake.1.gz
