--- ../snort3.orig/Makefile	2024-06-17 10:45:29.600043000 +0000
+++ Makefile	2024-06-17 16:50:40.866801000 +0000
@@ -21,6 +21,7 @@
 
 USES=		cmake compiler:c++14-lang cpe iconv localbase luajit pathfix pkgconfig \
 		shebangfix ssl
+DOCS_USES=	ruby:build
 PATHFIX_CMAKELISTSTXT=	create_pkg_config.cmake
 USE_GITHUB=	yes
 GH_ACCOUNT=	snort3
@@ -62,5 +63,8 @@
 STATICDAQ_CMAKE_BOOL=		ENABLE_STATIC_DAQ
 THREADSANITIZER_CMAKE_BOOL=	ENABLE_THREAD_SANITIZER
 TSC_CMAKE_BOOL=			ENABLE_TSC_CLOCK
+
+post-stage-DOCS-on:
+	${RUBY} ${WRKSRC}/doc/upgrade/get_differences.rb ${WRKSRC}/tools/snort2lua > ${STAGEDIR}${PREFIX}/share/doc/snort/config_changes.txt
 
 .include <bsd.port.mk>
