--- ../openssl32.orig/distinfo	2024-09-03 15:15:56.648923000 -0400
+++ distinfo	2024-09-03 15:10:04.856704000 -0400
@@ -1,3 +1,3 @@
-TIMESTAMP = 1717519504
-SHA256 (openssl-3.2.2.tar.gz) = 197149c18d9e9f292c43f0400acaba12e5f52cacfe050f3d199277ea738ec2e7
-SIZE (openssl-3.2.2.tar.gz) = 17744472
+TIMESTAMP = 1725390631
+SHA256 (openssl-3.2.3.tar.gz) = 52b5f1c6b8022bc5868c308c54fb77705e702d6c6f4594f99a0df216acf46239
+SIZE (openssl-3.2.3.tar.gz) = 17762604
