--- ../snort3.orig/files/patch-cmake_include__libraries.cmake	1970-01-01 00:00:00.000000000 +0000
+++ ./files/patch-cmake_include__libraries.cmake	2022-09-18 17:54:30.487643000 +0000
@@ -0,0 +1,10 @@
+--- cmake/include_libraries.cmake.orig	2022-09-18 17:43:24.402495000 +0000
++++ cmake/include_libraries.cmake	2022-09-18 17:44:20.140936000 +0000
+@@ -26,4 +26,6 @@
+ endif (ENABLE_SAFEC)
+ find_package(ICONV QUIET)
+ find_package(UUID QUIET)
+-find_package(Libunwind)
++if (ENABLE_LIBUNWIND)
++    find_package(Libunwind)
++endif (ENABLE_LIBUNWIND)
