--- ../snort3.orig/Makefile	2022-09-18 18:20:02.390353000 +0000
+++ ./Makefile	2022-09-18 18:08:21.450726000 +0000
@@ -32,7 +32,7 @@
 CONFLICTS=	snort-2*
 
 OPTIONS_DEFINE=		ADDRESSSANITIZER DEBUG DOCS FLATBUFFERS HYPERSCAN \
-			LARGEPCAP PIE STATICDAQ THREADSANITIZER TSC
+			LARGEPCAP LIBUNWIND PIE STATICDAQ THREADSANITIZER TSC
 OPTIONS_DEFAULT=	STATICDAQ
 OPTIONS_DEFAULT_amd64=	HYPERSCAN
 OPTIONS_SUB=		yes
@@ -41,6 +41,7 @@
 FLATBUFFERS_DESC=	Enable flatbuffers support
 HYPERSCAN_DESC=		Enable high-performance regular expression lib
 LARGEPCAP_DESC=		Enable support for pcaps larger than 2 GB
+LIBUNWIND_DESC=		Print stack trace in case of crash
 PIE_DESC=		Enable PIE (Broken)
 STATICDAQ_DESC=		Link static DAQ modules
 THREADSANITIZER_DESC=	Enable thread sanitizer
@@ -53,12 +54,17 @@
 HYPERSCAN_LIB_DEPENDS=		libhs.so:devel/hyperscan
 HYPERSCAN_PLIST_SUB=		HYPERSCAN=""
 HYPERSCAN_PLIST_SUB_OFF=	HYPERSCAN="@comment "
+LIBUNWIND_LIB_DEPENDS=		libunwind.so:devel/libunwind
+LIBUNWIND_CMAKE_BOOL=		ENABLE_LIBUNWIND
 LARGEPCAP_CMAKE_BOOL=		ENABLE_LARGE_PCAP
 PIE_CMAKE_BOOL=			ENABLE_PIE
 STATICDAQ_CMAKE_BOOL=		ENABLE_STATIC_DAQ
 THREADSANITIZER_CMAKE_BOOL=	ENABLE_THREAD_SANITIZER
 TSC_CMAKE_BOOL=			ENABLE_TSC_CLOCK
 
+DOCS_USE=			ruby=yes
+DOCS_VARS=			ruby_no_run_depends=yes
+
 .include <bsd.port.options.mk>
 
 .if ${ARCH:Mpowerpc64*}
@@ -74,5 +80,8 @@
 post-patch:
 	${REINPLACE_CMD} -e 's|_LIBDIR}/pkgconfig|_PREFIX}/libdata/pkgconfig|g' \
 		${WRKSRC}/cmake/create_pkg_config.cmake
+
+post-stage-DOCS-on:
+	${RUBY} ${WRKSRC}/doc/upgrade/get_differences.rb ${WRKSRC}/tools/snort2lua > ${STAGEDIR}${PREFIX}/share/doc/snort/config_changes.txt
 
 .include <bsd.port.mk>
