--- ../libpwquality.orig/Makefile	2022-09-15 21:23:48.886469000 +0000
+++ Makefile	2022-09-15 21:23:23.833158000 +0000
@@ -32,4 +32,8 @@
 PYTHON_CONFIGURE_ENABLE=	python-bindings
 PYTHON_CONFIGURE_ON=	--with-python-binary=${PYTHON_CMD}
 
+post-stage:
+	${CP} -R ${STAGEDIR}${PYTHON_SITELIBDIR}/${PKGNAME:S/^lib//}-py${PYTHON_VER}-${OPSYS:tl}-${_OSRELEASE}-${ARCH}.egg/EGG-INFO ${STAGEDIR}${PYTHON_SITELIBDIR}/${PKGNAME:S/^lib//}-py${PYTHON_VER}.egg-info
+	${CP} ${STAGEDIR}${PYTHON_SITELIBDIR}/${PKGNAME:S/^lib//}-py${PYTHON_VER}-${OPSYS:tl}-${_OSRELEASE}-${ARCH}.egg/${PORTNAME:S/^lib//}${PYTHON_EXT_SUFFIX}.so ${STAGEDIR}${PYTHON_SITELIBDIR}/
+
 .include <bsd.port.mk>
