--- ../py-CommonMark.orig/Makefile	2022-09-16 12:42:47.229023000 +0000
+++ Makefile	2022-09-16 12:41:44.747511000 +0000
@@ -14,11 +14,13 @@
 
 RUN_DEPENDS=	${PYTHON_PKGNAMEPREFIX}future>0:devel/py-future@${PY_FLAVOR}
 
-CONFLICTS_INSTALL=	cmark
-
 USES=		python:3.6+
 USE_PYTHON=	autoplist concurrent distutils
 
 NO_ARCH=	yes
+
+post-stage:
+	${MV} ${STAGEDIR}${PREFIX}/bin/cmark ${STAGEDIR}${PREFIX}/bin/cmark-py
+	${REINPLACE_CMD} -e 's=/bin/cmark$$=&-py=' ${WRKDIR}/.PLIST.*
 
 .include <bsd.port.mk>
