--- ../qt5-webengine.orig/Makefile	2023-03-10 18:43:33.759382000 +0000
+++ Makefile	2023-03-10 18:43:51.556868000 +0000
@@ -82,7 +82,7 @@
 # We pass `norecursive' to USES=qmake because src/plugins/plugins.pro checks
 # whether webenginewidgets is available, which fails when qmake processes all
 # .pro files at once.
-USES=		gl gnome gperf jpeg minizip ninja:build nodejs:build,lts \
+USES=		gl gnome gperf jpeg minizip ninja:build nodejs:build \
 		perl5 pkgconfig python:2.7,build qmake:norecursive,outsource \
 		qt-dist:5,webengine shebangfix xorg
 USE_GL=		gl
