--- ../kitty.orig/files/patch-kitty_gl.c	2023-04-20 04:00:48.871479000 +0000
+++ ./files/patch-kitty_gl.c	1970-01-01 00:00:00.000000000 +0000
@@ -1,11 +0,0 @@
---- kitty/gl.c.orig	2022-11-07 10:06:42 UTC
-+++ kitty/gl.c
-@@ -39,7 +39,7 @@ check_for_gl_error(void UNUSED *ret, const char *name,
- }
- 
- void
--gl_init() {
-+gl_init(void) {
-     static bool glad_loaded = false;
-     if (!glad_loaded) {
-         int gl_version = gladLoadGL(glfwGetProcAddress);
