--- ../kitty.orig/files/patch-kitty_cursor.c	2023-04-20 04:00:48.868691000 +0000
+++ ./files/patch-kitty_cursor.c	1970-01-01 00:00:00.000000000 +0000
@@ -1,11 +0,0 @@
---- kitty/cursor.c.orig	2022-11-07 10:06:42 UTC
-+++ kitty/cursor.c
-@@ -321,7 +321,7 @@ copy(Cursor *self, PyObject *a UNUSED) {
-     return (PyObject*)cursor_copy(self);
- }
- 
--Cursor *alloc_cursor() {
-+Cursor *alloc_cursor(void) {
-     return (Cursor*)new(&Cursor_Type, NULL, NULL);
- }
- 
